### PR TITLE
Convert to python

### PR DESCRIPTION
**Pull request by ghost**
A user encountered an issue where their tag did not conform to our preconceived notion of semver (vMAJOR.MINOR.PATCH). The auto-tagger did not handle this error and just told the user it created a new tag: "".

This inspired me to convert this script to python. There are a number of advantages to this:
- proper error handling
- use of modules (like [python semver](https://pypi.org/project/semver/)) which handle parsing wayyyyy better than my sorry ol' regex
- maintainability: other people know how to modify python better than odd bash scripts, surprisingly

### CODE REVIEW

File: entrypoint.py [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+    Returns:
+        None
+    """
+    url = f"https://api.github.com/repos/{os.getenv('GITHUB_REPOSITORY')}/issues/{pr_number}/comments"
+    headers = {
+        'Authorization': f"Bearer {os.getenv('GITHUB_TOKEN')}",
+        'Accept': 'application/vnd.github.v3+json'
+    }
+    body = {'body': comment_body}
+    requests.post(url, headers=headers, json=body)
```
Comment (rue-vmom) [inline entrypoint.py]: sucks that the `git` library doesn't support this.

**Review by rue-vmom — APPROVED** [review]
LGTM

File: entrypoint.py (hunk 1 above)
Comment (ghost) [reply entrypoint.py]: so, we could import the [PyGithub](https://pypi.org/project/PyGithub/) module which would likely be able to do this but given that its a single call and that is a decently heavy module, I don't think its worth it.

Comment (ghost) [reply entrypoint.py]: I lied, I wrote out what I think it would be, and I like it more.

**Pull request merged**